pkg/controller/modeladapter/utils_test.go

```diff
@@ -50,27 +49,6 @@ func TestEqualStringSlices(t *testing.T) {
 	})
 }
 
-// Test for getEnvKey function
-func TestGetEnvKey(t *testing.T) {
```

Comment:
this is very thin. It's just `os.LookUp` and we do not need the test coverage now.